crm-project/src/main/java/CRM/service/UserService.java

```diff
@@ -85,4 +88,17 @@ public List<User> getAllInBoard(long boardId) throws AccountNotFoundException {
         List<UserInBoard> usersInBoard = userInBoardRepository.findAllUserByBoard(board);
         return usersInBoard.stream().map(UserInBoard::getUser).collect(Collectors.toList());
     }
+
+    public UserInBoardDTO addUserToBoard(long userId, long boardId) throws AccountNotFoundException {
```

Comment:
Service dosent suppose to know about DTO's
the conversion from an entity to entityDTO happens in the Facade

Reply:
Got you.